README.md

```diff
@@ -1,37 +1,123 @@
-#WP PHP Console
```

Comment:
I tried to stick to these names in README.md and README.txt
* PHP Console (by barbushin)
* PHP Console server library
* Chrome plugin PHP Console
* WP PHP Console (WP Plugin)
* Remote PHP Eval Terminal
* JavaScript console
* PHP Console eval & options form